src/components/dashboard.jsx

```diff
@@ -12,12 +12,24 @@ import CurrentUserStore from '../user-store';
 import AsyncButton from './async-button';
 import Time from './time';
 
-const SAMPLE_REPOS = [
+let SAMPLE_REPOS = [
   {repoOwner: 'huboard', repoName: 'huboard'},
   {repoOwner: 'openstax', repoNames: ['tutor-js', 'tutor-server'], comment: ' (multiple repositories)'},
   {repoOwner: 'jquery', repoName: 'jquery'}
 ];
 
+try {
+  const repos = REPOSITORIES;
+  if (repos) {
+    const repoOwner = repos.split(':')[0];
+    const repoNames = repos.substring(repos.indexOf(':') + 1).split('|');
+    SAMPLE_REPOS.push({
+      repoOwner, repoNames
+    });
+  };
+} catch (error) {
+  console.log('warning:', error);
```

Comment:
Give a single whitespace after `warning:`.

Reply:
Isn't that done by `console.log` automatically?

Reply:
Ah, you were right. I just see it passed on the second parameter.